website/source/layouts/docs.erb

```diff
@@ -141,6 +141,19 @@
 
 					<li<%= sidebar_current("docs-commands-operator") %>>
 					<a href="/docs/commands/operator.html">operator</a>
+					<ul class="subnav">
```

Comment:
Move the bit on the raft subcommand page to the operator one, and show `raft list-peers` and `raft remove-peer` instead for the sidebar?